src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.h

```diff
@@ -6,6 +6,15 @@
 
 struct ObjMakeoshihiki;
 
+typedef struct {
+    /* 0x00 */ Vec3f posVecs[3];
+    /* 0x24 */ u8 unk_24[0x3];
+    /* 0x27 */ u8 paramVal1;
+    /* 0x28 */ u8 paramVal2;
+    /* 0x29 */ char unk_29[0x1];
+    /* 0x2A */ s16 rotY;
+} BlockConfig;
```

Comment:
is this going to be used externally by other actors? if not it should stay in c

Reply:
Yeah I was unsure, but I'll move it back.